### ISSUE TITLE
Feature request: A Layer object wrapping multiple Layer objects

### ISSUE THREAD

**Issue by varshiths** [tensorflow/tensorflow#20399]
`tf.contrib.seq2seq.BasicDecoder` module allows provision for a single output layer, to modify the output of the RNN before proceeding to further sample.

In my particular application, I require application of multiple layers.
This is not possible as far as I have checked.

A `MultiLayer` class could be implemented, similar to the `tf.contrib.rnn.MultiRNNCell`.
This class could wrap multiple `Layer` objects and apply them on `call`.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by varshiths** [comment]
Have I written custom code - N/A
OS Platform and Distribution - N/A
TensorFlow installed from - N/A
TensorFlow version - 1.8
Bazel version - N/A
CUDA/cuDNN version - N/A
GPU model and memory - N/A
Exact command to reproduce - N/A

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by bignamehyp** [comment]
We will consider it, thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 46 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 61 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 76 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 91 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by rmothukuru** [comment]
@varshiths,
Sorry for the delayed response. Can you please let us know if adding [StackedRNNCells](https://www.tensorflow.org/api_docs/python/tf/keras/layers/StackedRNNCells) as an [Argument, Output_Layer](https://www.tensorflow.org/addons/api_docs/python/tfa/seq2seq/BasicDecoder#args) in the [BasicDecoder](https://www.tensorflow.org/addons/api_docs/python/tfa/seq2seq/BasicDecoder) fulfills your request? Thanks!
